bundle/manifests/limitador-operator.clusterserviceversion.yaml

```diff
@@ -37,7 +37,6 @@ metadata:
     capabilities: Basic Install
     categories: Integration & Delivery
     containerImage: quay.io/kuadrant/limitador-operator:latest
-    createdAt: "2024-05-16T14:52:42Z"
```

Comment:
do not remove `createdAt` field.

Reply:
Oops i didnt notice ill add it back

Reply:
This was a issue in the end with the make bundle removing it due to versioning should be fixed with https://github.com/Kuadrant/limitador-operator/pull/148